cmd/fuzzbinder/main.go

```diff
@@ -42,6 +42,8 @@ func main() {
 		"Required - Fuzzing date. The expected date format is YYYYMMDD.")
 	fuzzClaimPath := flag.String("fuzzclaim_path", "fuzzclaim.json",
 		"Optional - Output file name for storing the generated fuzzing claim.")
+	validityDays := flag.Int("validity", 7,
```

Comment:
I am not sure if number of days is the most convenient way of getting the validity range. How about directly getting `notBefore` and `notAfter` dates as input? We could use today's date as the default value for `notBefore`.

Reply:
-  I agree that having `notAfter` and  `notBefore` as inputs is simpler for the user (not really for us, especially for `notBefore` ). 
Indeed, this can lead to some reliability problems: 
-  `notBefore` it should be verified to check it is set after the time of generating the claim to avoid tempering with the FuzzClaim. 
-  In the current version where `notBefore` is fixed, the exact time of claim generation can be found easily and reliably. This makes it easier to reproduce claims (since it means that only logs prior to that exact time were used). If the `notBefore` is an entry, this can create problems for reproducing claims, especially when logs are still being generated on a given day and FuzzBinder generates the fuzzClaim before the end of the day.
-  `notAfter` should be after `notBefore`

Reply:
I suggest in that case to keep `notBefore` fixed by FuzzBinder and only allow `notAfter` to be an input.

Reply:
Generally the contract for ClaimValidity is that `notBefore` is the same time or after the time the issue was generated. So I still think `notBefore` should be an input, but it has to be validated to be after the current time. 

For reproducibility, the date for which the claim was generated should be included in the FuzzClaimSpec. In particular, we should not use `fuzzParameters.Date` as the `notBefore` date.

Reply:
Oh, there is already the `IssuedOn` for that. I forgot about it.

Reply:
Oh! Yes 👍🏼

Reply:
Just to be clear, `IssuedOn`, `NotBefore`, and `FuzzParameters.Date` could be the same date, but they don't have to be. The invariant that we should maintain is `FuzzParameters.Date <= IssuedOn <= NotBefore < NotAfter`.

Reply:
I think that we have it now since 
- we validate that `CurrentTime = IssuedOn <= NotBefore < NotAfter` in the validity verification
- we validate `FuzzParameters.Date <= IssuedOn` when we validate the fuzzing date

What we have now: `<= IssuedOn -15 <= FuzzParameters.Date <= IssuedOn <= NotBefore < NotAfter`

Reply:
The current problem is that it is hard to have `IssuedOn = NotBefore` since `NotBefore` comes from an input in `YYYYMMDD` format (without the exact time) while  `IssuedOn` is an exact time.

Reply:
That is fine. We don't need to have `IssuedOn = NotBefore`. But if it happens in any way, it does not violate any rule.